src/AzureIoTHub.Portal.Client/Pages/EdgeModels/EdgeModule/ModuleDialog.razor

```diff
@@ -104,6 +120,10 @@
         Module.ModuleName = currentModuleName;
         Module.ImageURI = currentImageUri;
         Module.ContainerCreateOptions = currentContainerCreateOptions;
+
+        if (Portal.CloudProvider.Equals(CloudProviders.Azure)) { Module.Version = " "; }
```

Comment:
Why setting a value with a space for Azure?

Reply:
Because I put the version field as a required in IotEdgeModel (in Models)